requirements-dev.txt

```diff
@@ -1,34 +1,60 @@
-Pygments==2.15.1
-black==23.3.0
+Flask==2.3.2
+Jinja2==3.1.2
+MarkupSafe==2.1.3
+PyYAML==6.0.1
+Pygments==2.16.1
+SecretStorage==3.3.3
+Werkzeug==2.3.6
+attrs==23.1.0
+black==23.7.0
 bleach==6.0.0
+blinker==1.6.2
+brotlicffi==1.0.9.2
 build==0.10.0
-click==8.1.3
+cffi==1.15.1
+click==8.1.6
 coverage==7.2.7
+cryptography==41.0.3
+decorator==5.1.1
 docutils==0.20.1
-importlib-metadata==6.7.0
+flasgger==0.9.7.1
+gevent==23.7.0
+greenlet==2.0.2
+httpbin==0.10.0
+importlib-metadata==6.8.0
 iniconfig==2.0.0
 isort==5.12.0
-jaraco.classes==3.2.3
-keyring==23.13.1
+itsdangerous==2.1.2
+jaraco.classes==3.3.0
+jeepney==0.8.0
+jsonschema-specifications==2023.7.1
+jsonschema==4.19.0
+keyring==24.2.0
 markdown-it-py==3.0.0
 mdurl==0.1.2
-more-itertools==9.1.0
+mistune==3.0.1
+more-itertools==10.1.0
 mypy-extensions==1.0.0
 packaging==23.1
-pathspec==0.11.1
+pathspec==0.11.2
 pkginfo==1.9.6
-platformdirs==3.6.0
-pluggy==1.1.0
-pyflakes==3.0.1
+platformdirs==3.10.0
+pluggy==1.2.0
+pycparser==2.21
+pyflakes==3.1.0
 pyproject_hooks==1.0.0
 pytest-cov==4.1.0
-pytest-network==0.0.1
-pytest==7.3.2
+pytest-httpbin==2.0.0
+pytest==7.4.0
 readme-renderer==40.0
+referencing==0.30.2
 requests-toolbelt==1.0.0
 rfc3986==2.0.0
-rich==13.4.2
+rich==13.5.2
+rpds-py==0.9.2
 six==1.16.0
 twine==4.0.2
 webencodings==0.5.1
-zipp==3.15.0
+zipp==3.16.2
+zope.event==5.0
+zope.interface==6.0
```

Comment:
These updated with `./script/update-requirements`? When I run it locally on this branch I get a few diffs:

```console
coho:requests-futures ross$ git diff
diff --git a/requirements-dev.txt b/requirements-dev.txt
index 330d5b6..c652801 100644
--- a/requirements-dev.txt
+++ b/requirements-dev.txt
@@ -3,7 +3,6 @@ Jinja2==3.1.2
 MarkupSafe==2.1.3
 PyYAML==6.0.1
 Pygments==2.16.1
-SecretStorage==3.3.3
 Werkzeug==2.3.6
 attrs==23.1.0
 black==23.7.0
@@ -14,7 +13,6 @@ build==0.10.0
 cffi==1.15.1
 click==8.1.6
 coverage==7.2.7
-cryptography==41.0.3
 decorator==5.1.1
 docutils==0.20.1
 flasgger==0.9.7.1
@@ -26,7 +24,6 @@ iniconfig==2.0.0
 isort==5.12.0
 itsdangerous==2.1.2
 jaraco.classes==3.3.0
-jeepney==0.8.0
 jsonschema-specifications==2023.7.1
 jsonschema==4.19.0
 keyring==24.2.0
```

Reply:
Doesn't appear to be:

```console
(env) coho:requests-futures ross$ python --version
Python 3.11.4
```

Maybe OS. I'm on OSX. Anyway. Just seemed like odd requirements, but probably not hurting anything, I guess.

Reply:
Ah yes, it's probably because I'm running on Linux ;-).